dataeng/jobs/createJobs.groovy

```diff
@@ -81,6 +82,7 @@ def taskMap = [
     GENERATE_WAREHOUSE_DOCS_JOB: GenerateWarehouseDocsJob,
     SQL_SCRIPTS_JOB: SqlScriptsJob,
     SINGLE_SQL_SCRIPT_JOB: SingleSqlScriptJob,
+    LOAD_AFFILIATE_WINDOW_JOB: LoadAffiliateWindowWarehouseJob,
```

Comment:
I think you'll probably also want to add `name('affiliate-window')` to the `listView('Warehouse') {` block below so that the job shows up under the "Warehouse" tab in the jenkins UI.